sormas-rest/pom.xml

```diff
@@ -47,19 +47,21 @@
 			<artifactId>jersey-media-json-jackson</artifactId>
 		</dependency>
 
-	  	<dependency>
+		<dependency>
 			<groupId>javax.security.enterprise</groupId>
 			<artifactId>javax.security.enterprise-api</artifactId>
 		</dependency>
 
 		<dependency>
 			<groupId>io.swagger.core.v3</groupId>
 			<artifactId>swagger-jaxrs2</artifactId>
+			<version>2.2.3</version>
```

Comment:
Inherit the version tag.

Reply:
Done ([cf5b813](https://github.com/hzi-braunschweig/SORMAS-Project/pull/11202/commits/cf5b813abb3a973f6df3d1eb8229bd8cbeecb9a8))